.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*.md]
+end_of_line = lf
+trim_trailing_whitespace = true
+
+[*.rb]
+end_of_line = lf
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2
```

Comment:
We really probably should have a global standard .editorconfig for the whole Exercism project and every repo just uses the same copy.  Thoughts?

Reply:
I'd suggest maybe building on the one in exercism/v3 for now? Get that agreed across tracks, then we can port it.